Sources/ODP/OdpManager.swift

```diff
@@ -80,6 +81,12 @@ public class OdpManager {
                                                completionHandler: completionHandler)
     }
     
+    func sendInitializedEvent(vuid: String) throws {
+        guard enabled else { throw OptimizelyError.odpNotEnabled }
+        guard odpConfig.eventQueueingAllowed else { throw OptimizelyError.odpNotIntegrated }
```

Comment:
Does this throw errors for default action (initailized) when ODP is not used? Can you double check that clients not using ODP not see any error messages from this?